websites.yml

```diff
@@ -68,10 +73,15 @@ avatar:
   github: opengraph
   gitlab: opengraph
   instagram: opengraph
+  opensuse: false
   openhub: false
   pinterest:
     key: image_xlarge_url
   soundcloud: opengraph
   tumblr: opengraph
   twitter:
     url: https://twitter.com/{u}/profile_image?size=original
+phrase:
+  opensuse: "About me"
+  pinterest: "{u}"
+  gitlab: "@{u}"
```

Comment:
I think we can have `{u}` for gitlab as well. No need for `@`

Also, to make it clearer, I'd prefer having `{username}` here.

Reply:
another thought, shouldn't we put phrase key under these websites rather than creating a separate section for it. 
cc: @jayvdb

Reply:
I put the @ in because {u} didn't work on it's own. phrase is separate because the way the other sections are named, putting a phrase in them would be confusing, but up to @jayvdb to decide.

Reply:
If we put it under `urls`, then they would need to change structure from `<website>: <url>` to
```
urls:
  <website>:
    url: <url>
    confirmation_phrase: {u}
```

Not sure about that.

Reply:
makes sense. we can preserve this one then

Reply:
We could do something funky like use a hash fragment to store the string ; i.e. http://www.gerv.net/software/fragment-search/
e.g.
`<website>: <url>#!s!{u}`

Reply:
If we keep this section, which IMO is good enough, then it should be moved under `urls` as it is directly related to the username urls for the websites.

It should also be renamed to be more explicit.  I suggest something like `content_verification` or `HTTP404_emulation_phrase`

Reply:
@manu-chroma surely {u} is clearer than {username} as that's what is being used in the urls section? just my opinion, I don't know

Reply:
It is obvious in the urls. I thought it might not be when coming to the ``content_verification`` section. I don't mind having ``{u}`` :smiley:

Reply:
@manu-chroma okay, I moved the section underneath urls so it should be fine,
if that's all issues resolved, can you merge?

Reply:
> We could do something funky like use a hash fragment to store the string ; i.e. http://www.gerv.net/software/fragment-search
e.g.
<website>: <url>#!s!{u}

will read up on this :P